website/docs/reference/project-configs/macro-paths.md

```diff
@@ -16,7 +16,26 @@ macro-paths: [directorypath]
 Optionally specify a custom list of directories where [macros](/docs/build/jinja-macros#macros) are located. Note that you cannot co-locate models and macros.
 
 ## Default
-By default, dbt will search for macros in a directory named `macros`, i.e. `macro-paths: ["macros"]`
+By default, dbt will search for macros in a directory named `macros`, i.e. `macro-paths: ["macros"]`. 
+
+import RelativePath from '/snippets/_relative-path.md';
+
+<RelativePath 
+path="macro-paths"
+absolute="/Users/username/project/macros"
+/>
+
+- ✅ **Do:**
+    ```yml
+    # Recommended relative path example
```

Comment:
Can we put "Recommended relative path example" and "Avoid using absolute paths" where "Do" and "Don't" are?